packages/stateless/components/NftCard.tsx

```diff
@@ -262,17 +265,37 @@ export const NftCard = forwardRef<HTMLDivElement, NftCardProps>(
               </div>
             )}
 
-            {/* Floor price */}
-            {floorPrice && (
+            {/* Highest offer */}
+            {highestOffer && (
               <div className="space-y-2">
-                <p className="secondary-text">{t('title.floorPrice')}</p>
+                <p className="secondary-text">{t('title.highestOffer')}</p>
+                <div className="body-text font-mono">
+                  {highestOffer.amount && highestOffer.offerToken && (
```

Comment:
```suggestion
                  {typeof highestOffer.amount === 'number' && highestOffer.offerToken && (
```

if amount is `0`, it is falsey but react will still render a `0`, so an explicit numeric check is better here